src/electron/start.action.mjs

```diff
@@ -26,11 +26,9 @@ import {spawnStream} from '../build/spawn_stream.mjs';
  * @param {string[]} parameters
  */
 export async function main(...parameters) {
-  const {platform, buildMode} = getElectronBuildParameters(parameters);
+  const {buildMode} = getElectronBuildParameters(parameters);
 
-  await runAction('www/build', platform, `--buildMode=${buildMode}`);
```

Comment:
No, this just makes the build very slow (~5 min) because we invoked the same script for more than 3 times. I can put them back if it conflicts with the future design.

Reply:
I added them back, we can discuss about it in your PR.